README.md

```diff
@@ -1,3 +1,98 @@
-# Triton-optimized Couchbase
+# Autopilot pattern for Couchbase
```

Comment:
I've been using the former because I think the latter sounds vaguely awkward. But I'm fine with whichever we decide is the correct usage.